src/interval.rs

```diff
@@ -71,6 +72,15 @@ impl Stream for Interval {
     }
 }
 
+impl fmt::Debug for Interval {
```

Comment:
I think this can also be replaced w/#[derive(Debug)] since `Delay` implements it directly and `Duration` likely does too.